# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wxwidgets/pango.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ur wxPython-src-2.6.1.0_orig/src/gtk/dcclient.cpp wxPython-src-2.6.1.0/src/gtk/dcclient.cpp
--- wxPython-src-2.6.1.0_orig/src/gtk/dcclient.cpp	2005-05-23 19:23:47.000000000 +0200
+++ wxPython-src-2.6.1.0/src/gtk/dcclient.cpp	2005-09-11 00:55:49.000000000 +0200
@@ -1883,9 +1883,9 @@
 
             // We might want to use the X11 context for faster
             // rendering on screen
-            if (m_font.GetNoAntiAliasing())
-                m_context = m_owner->GtkGetPangoX11Context();
-            else
+	    //if (m_font.GetNoAntiAliasing())
+            //    m_context = m_owner->GtkGetPangoX11Context();
+            //else
                 m_context = m_owner->GtkGetPangoDefaultContext();
 
             // If we switch back/forth between different contexts
diff -ur wxPython-src-2.6.1.0_orig/src/gtk/window.cpp wxPython-src-2.6.1.0/src/gtk/window.cpp
--- wxPython-src-2.6.1.0_orig/src/gtk/window.cpp	2005-06-02 20:29:40.000000000 +0200
+++ wxPython-src-2.6.1.0/src/gtk/window.cpp	2005-09-11 01:39:14.000000000 +0200
@@ -4281,10 +4281,7 @@
 
 PangoContext *wxWindowGTK::GtkGetPangoX11Context()
 {
-    if (!m_x11Context)
-        m_x11Context = pango_x_get_context( gdk_display );
-
-    return m_x11Context;
+    return gtk_widget_get_pango_context(m_widget);
 }
 #endif
  
